ERROR - 2024-06-28 00:04:54 --> mysqli_sql_exception: Duplicate entry '1' for key 'clienteid' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `ve...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ve...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ve...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ve...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\almacen\system\Model.php(791): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\almacen\app\Controllers\Ventas.php(75): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Ventas->create()
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
#10 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-06-28 00:04:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'clienteid'
[Method: POST, Route: ventas]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `venta` (`clienteid`, `fechaventa`) VALUES (:clienteid:, :fechaventa:)', [...], false)
 2 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Ventas.php(75): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->create()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-28 00:04:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'clienteid'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `venta` (`clienteid`, `fechaventa`) VALUES (\'1\', \'111\')')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `venta` (`clienteid`, `fechaventa`) VALUES (\'1\', \'111\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `venta` (`clienteid`, `fechaventa`) VALUES (:clienteid:, :fechaventa:)', [...], false)
 4 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Ventas.php(75): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-28 00:04:54 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'clienteid'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('INSERT INTO `venta` (`clienteid`, `fechaventa`) VALUES (\'1\', \'111\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `venta` (`clienteid`, `fechaventa`) VALUES (\'1\', \'111\')')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `venta` (`clienteid`, `fechaventa`) VALUES (\'1\', \'111\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2325): CodeIgniter\Database\BaseConnection->query('INSERT INTO `venta` (`clienteid`, `fechaventa`) VALUES (:clienteid:, :fechaventa:)', [...], false)
 5 SYSTEMPATH\Model.php(383): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(791): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Ventas.php(75): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->create()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-28 00:05:40 --> mysqli_sql_exception: Unknown column 'venta.idventa' in 'where clause' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
#6 C:\xampp\htdocs\almacen\app\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('1')
#7 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Ventas->update('1')
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-06-28 00:05:40 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'venta.idventa' in 'where clause'
[Method: PUT, Route: ventas/1]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 2 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 4 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('1')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('1')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-28 00:05:40 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'venta.idventa' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 4 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 6 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('1')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('1')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-28 00:05:40 --> [Caused by] mysqli_sql_exception: Unknown column 'venta.idventa' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 5 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '1')
 7 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('1')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('1')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-28 00:05:54 --> mysqli_sql_exception: Unknown column 'venta.idventa' in 'where clause' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, '0')
#6 C:\xampp\htdocs\almacen\app\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('0')
#7 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Ventas->update('0')
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-06-28 00:05:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'venta.idventa' in 'where clause'
[Method: PUT, Route: ventas/0]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 2 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '0')
 4 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('0')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('0')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-28 00:05:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'venta.idventa' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 4 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '0')
 6 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('0')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('0')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-28 00:05:54 --> [Caused by] mysqli_sql_exception: Unknown column 'venta.idventa' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 5 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '0')
 7 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('0')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('0')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-06-28 00:10:37 --> mysqli_sql_exception: Unknown column 'venta.idventa' in 'where clause' in C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php:308
Stack trace:
#0 C:\xampp\htdocs\almacen\system\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\almacen\system\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\almacen\system\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\almacen\system\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\almacen\system\BaseModel.php(610): CodeIgniter\Model->doFind(true, '0')
#6 C:\xampp\htdocs\almacen\app\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('0')
#7 C:\xampp\htdocs\almacen\system\CodeIgniter.php(933): App\Controllers\Ventas->update('0')
#8 C:\xampp\htdocs\almacen\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
#9 C:\xampp\htdocs\almacen\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\almacen\system\Boot.php(312): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\almacen\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\almacen\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2024-06-28 00:10:37 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'venta.idventa' in 'where clause'
[Method: PUT, Route: ventas/0]
in SYSTEMPATH\Database\BaseConnection.php on line 676.
 1 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 2 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '0')
 4 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('0')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('0')
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-28 00:10:37 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'venta.idventa' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 313.
 1 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'')
 2 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 4 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '0')
 6 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('0')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('0')
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-06-28 00:10:37 --> [Caused by] mysqli_sql_exception: Unknown column 'venta.idventa' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 308.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(308): mysqli->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(722): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'')
 3 SYSTEMPATH\Database\BaseConnection.php(636): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = \'0\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1629): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `venta`
WHERE `venta`.`idventa` = :venta.idventa:', [...], false)
 5 SYSTEMPATH\Model.php(207): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(610): CodeIgniter\Model->doFind(true, '0')
 7 APPPATH\Controllers\Ventas.php(113): CodeIgniter\BaseModel->find('0')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ventas->update('0')
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ventas))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(312): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
